pycbc/workflow/pegasus_files/osg-testing-template.xml

```diff
@@ -0,0 +1,15 @@
+  <site handle="osg-testing" arch="x86_64" os="LINUX">
+    <profile namespace="env" key="GLOBUS_LOCATION">/usr</profile>
+    <profile namespace="pegasus" key="style">condor</profile>
+    <profile namespace="condor" key="getenv">True</profile>
+    <profile namespace="condor" key="accounting_group">$ACCOUNTING_GROUP</profile>
+    <profile namespace="condor" key="should_transfer_files">YES</profile>
+    <profile namespace="condor" key="when_to_transfer_output">ON_EXIT_OR_EVICT</profile>
+    <profile namespace="condor" key="+DESIRED_Sites">&quot;UCSD,Nebraska,SU-OG,GATech,Wisconsin,Omaha&quot;</profile>
```

Comment:
For `osg-testing`, why not completely skip the whitelist and accept all sites that have been added to GlideinWMS?